scripts/ebs-warmer.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Source: https://gist.github.com/muhqu/3293988
+# Ref: https://forums.aws.amazon.com/message.jspa?messageID=198413
+
+if [ -z "$1" ]; then
+    echo "Usage: sudo $0 /dev/sdh1"
+    exit 1;
+fi
+
+dd if=$1 of=/dev/null & pid=$!
```

Comment:
Good catch. By the way, I decided to use this helper script instead of just running `dd` inside the cloud config, so that we can monitor the progress and benchmark how long the process takes.